_posts/en/newsletters/2019-05-29-newsletter.md

```diff
@@ -0,0 +1,566 @@
+which each paid one of the receivers---but she wouldn't need to form
+this into a specific distribution transaction.  Instead, she'd just hash
+the ten outputs together and use the resultant digest to create a
+tapleaf script containing COSHV.  That would be the only tapleaf in this
+Taproot commitment.  Alice could also use the participants' public keys
+to form the taproot internal key to allow them to cooperatively spend
+the money without revealing the Taproot script path.
+
+![A setup transaction paying a COSHV output that expands into a distribution transaction](/img/posts/2019-05-tx-tree-2.png)
+
+Alice would then give each of the receivers a copy of all ten outputs to
+allow each of them to verify that Alice's setup transaction,
+when suitably confirmed, guaranteed them payment.  When they later
+wanted to spend that payment, any of them could then create a distribution
+transaction containing the committed outputs.  Unlike the example from
+the previous subsection, they don't need to pre-sign anything so they would never need to interact with each
+other.  Even better, the information Alice needs to send them in order
+to allow them to verify the setup transaction and ultimately spend their
+money could be sent through existing asynchronous communication methods
+such as email or a cloud drive.  That means the receivers wouldn't need
+to be online at the time Alice created and sent her setup transaction.
+
+This elimination of the need to interact is a particular highlight of
+the proposal.  If we imagine the example above with Alice being an
+exchange, the interactive form of the protocol would require her to keep
+the ten participants online and connected to her service from the moment
+each of them submitted their withdrawal request until the interaction
+was done---and they'd all need to use wallets compatible with such a
+child transaction signing protocol.  The non-interactive form with COSHV
+would only require them to submit a Bitcoin address and an email address
+(or some other protocol address for delivery of the committed outputs).
+
+### More advanced usage
+
+In our previous examples, getting the distribution transaction confirmed
+was an all-or-nothing endeavor.  Either all ten receivers finally
+received their money at the same time, or all of them were still waiting
+for that confirmation to occur.  However, with Taproot and COSHV,
+it's possible to allow each receiver to individually withdraw their
+money from the covenant while also allowing most the other receivers to
+keep their money offchain for the moment.
+
+For example, let's imagine Alice paid each of our ten receivers 1 BTC
+using a tree of outputs where each output paid two or three other outputs until
+an individual's money was withdrawn from the covenant.  A complete tree
+might for ten participants might look like this:
+
+![Example output tree for ten participants](/img/posts/2019-05-tx-tree-5.png)
```

Comment:
It's not clear to me that you want to have a 'bushy tree' which pays out along a path rather than a simplest possible 8 or 16 node binary tree where payouts only happen on the leaf nodes. But you can, so it's not inaccurate. There's a trade off around free-riding and other things.

But I'm OK with is as the example of how to do it.